### ISSUE TITLE
Add benchmarks for querying basic nfts

### ISSUE THREAD

**Issue by ethanfrey** [iov-one/weave#235]
This is a continuation of #224 

This may involve adding some secondary indexes to the existing tokens if they don't support the queries.

We want to try the following queries:
* username: query by username, by owner, by chain-address pair (both registered and not registered)

These can be moved into another issue to be done much later:
* blockchain: query by chain id , query by owner (also primary key if that should be different than chainid)
* bootstrap_node: by owner, by blockchain it is associated with (eg. list all bootstrap nodes form this blockchain_id), also by id as baseline
* ticker: query by ticker name, by owner, by blockchain it is associated with (eg. list all tickers registered on a given blockchain_id)

**Comment by ethanfrey** [comment]
Update. After #282 only the username nft is priority to be benchmarked. Other benchmarks can be done much later (when we start on namespace implementation)

**Comment by ethanfrey** [comment]
I don't know if we need any benchmarks after all....
